File: 3/conditionals.clj
(use '[clojure.string :only (split triml)])

(let [N_t (read-line) 
      N (Integer/parseInt N_t)
      is_odd (= 1 (mod N 2))
      ]
  (println (cond is_odd "Weird"
                 (<= N 5) "Not Weird"
                 (<= N 20) "Weird"
                 :else "Not Weird")))

File: 3/conditionals.js
process.stdin.resume();
process.stdin.setEncoding('ascii');

var input_stdin = "";
var input_stdin_array = "";
var input_currentline = 0;

process.stdin.on('data', function (data) {
    input_stdin += data;
});

process.stdin.on('end', function () {
    input_stdin_array = input_stdin.split("\n");
    main();    
});

function readLine() {
    return input_stdin_array[input_currentline++];
}

/////////////// ignore above this line ////////////////////

function main() {
  var N = parseInt(readLine());
  var statement = "";
  if (N % 2 == 0) {
    if (N <= 5) {
      statement = "Not Weird"
    } else if (N <= 20) {
      statement = "Weird"
    } else {
      statement = "Not Weird"
    }
  } else {
    statement = "Weird"
  }
  process.stdout.write(statement)

}
